Pidaq/UI/README.md

```diff
@@ -15,19 +15,19 @@ Contains dash control GUIs for each project, as well as grafana dashbooard .json
 Using DTS as an example, however they will all follow a similar structure
 ```
 DTS  
-├── dts.py                      # Application entrypoint, used to launch the dash app   
+├── dts-dash.py                      # Application entrypoint, used to launch the dash app   
 ├── dts_dash/                   # Python Package containing all dash layout and callbacks for the dts dash app  
     ├── __init__.py                   
     ├── app.py                      # The dash app that gets imported by callback modules and dts.py  
     ├── assets/                     # Contains bootstrap css framework, and one custom .css file  
         ├── css
-        ├── js                            
+        ├── js
+        ├── img                       
```

Comment:
Should probably sort alphabetically... Not sure what the standard is for directory diagrams